lua/dashboard/theme/hyper.lua

```diff
@@ -101,7 +101,7 @@ local function project_list(config, callback)
     limit = 8,
     icon = ' ',
     icon_hl = 'DashboardRecentProjectIcon',
-    action = 'Telescope find_files cwd=',
+    action = function(path) vim.cmd("Telescope find_files cwd=" .. path) end,
```

Comment:
why changed here?

Reply:
this code can restored, it doesn't matter. i just write a sample for support function

Reply:
no need changed here

Reply:
ok, i restore it